control/control.yaml

```diff
@@ -48,6 +51,20 @@ components:
         state:
           description: >-
             The transmit state.
+
+            If the value of the state property is 'start' then all flows
+            that are included in the 'flow_names' property MUST have their
+            metric counters cleared.
+
+            If the value of the state property is 'pause' then all flows that
+            are included in the 'flow_names' property will be stopped but
```

Comment:
Perhaps say "included in the  'flow_names' property will be paused but their metric..."?

Also, do we want to distinguish between resuming a paused flow and starting a stopping flow? ie, the available enums are "[start, stop, pause, resume]"?

Reply:
Instead of adding more options I am proposing to replace pause with an state option of 'start_with_no_clear'.  This simplifies the setting of state and explicitly calls out the not clearing of metric counters.